wikimedia/wikipedia-ios
# offset for find & replace adjusted content inset bugs

https://phabricator.wikimedia.org/T219888

I tested in a sample project that has no content inset adjustments and it seems the system sometimes accounts for the keyboard height in adjustedContentInset and sometimes doesn't. It does this regardless of what value is set in contentInsetAdjustmentBehavior on the scroll view. This makes me think that refactoring to additionalSafeAreaInsets may not work. This is my low-risk attempt to offset for it for 6.3.1 but it does cause too much spacing on the bottom in some instances. IMO too much spacing is better than too little but we can discuss it.

Test steps are:
1. Make find or replace textfield first responder > rotate > scroll and make sure you can see all content (iPhone & iPad).
2. Make find or replace textfield first responder > tap 3 dots > tap an option on action sheet in iPhone > confirm you can see all content after action sheet option was selected.
